src/applications/vaos/new-appointment/components/TypeOfCarePage/index.jsx

```diff
@@ -134,39 +128,8 @@ function TypeOfCarePage({
       )}
       <PodiatryAppointmentUnavailableModal
         showModal={showPodiatryApptUnavailableModal}
-        onClose={hidePodiatryAppointmentUnavailableModal}
+        onClose={dispatch(hidePodiatryAppointmentUnavailableModal)}
```

Comment:
I think this needs to be `() => dispatch(hidePodiatryAppointmentUnavailableModal())`